### PR TITLE
GET comments

### PR DESCRIPTION
**Pull request by proyectoramirez**
These changes add a list of greetings to the Java servlet, which are requested and rendered by the website.

Demo: https://comment-servlet-dot-josedramirez-step-2020.appspot.com/

This should be merged after #3.

### CODE REVIEW

**Comment by proyectoramirez** [comment]
Opened new pull request. It now only includes the new changes to master.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+*.swp
+*~.nib
+local.properties
+.loadpath
+.factorypath
+
+##############################
+## VSCode
+##############################
+.vscode/
```
Comment (Enriqe) [inline .gitignore]: nit: newline at EOF

**Review by Enriqe — COMMENTED** [review]
Can you add the link of the demo to the description please?

File: .gitignore (hunk 1 above)
Comment (proyectoramirez) [reply .gitignore]: Done in latest commit

**Comment by proyectoramirez** [comment]
Added link to description!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -295,10 +295,48 @@ function buildSectionEntryDescription(description) {
...
+ */
+function buildGreetingFragment(greeting) {
+  const fragment = document.createElement("div");
+  fragment.append(greeting);
+
+  return fragment;
+}
+
+/**
+ * Requests a list of greetings from "/data".
```
Comment (Enriqe) [inline portfolio/src/main/webapp/script.js]: No need to say where it comes from, just a general comment about what the function does is enough. Being too specific in the JSDoc makes it hard to maintain (having to change it every time you change the endpoint, e.g.)

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -295,10 +295,48 @@ function buildSectionEntryDescription(description) {
...
+
+  mainContainer.prepend(...greetingFragments);
+}
+
+/**
+ * Creates and populates a greeting fragment.
+ * 
+ * @param {string} greeting The greeting to populate the fragment.
+ * 
+ * @return {!HTMLElement} The populated fragment.
```
Comment (Enriqe) [inline portfolio/src/main/webapp/script.js]: change to `!Element`

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (proyectoramirez) [reply portfolio/src/main/webapp/script.js]: Gotcha, updated!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (proyectoramirez) [reply portfolio/src/main/webapp/script.js]: Done in latest commits

**Review by Enriqe — APPROVED** [review]
👍

**Pull request merged**